keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ArtifactHandler.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.actuation
+
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+
+interface ArtifactHandler {
+  val name: String
+    get() = javaClass.simpleName
+
+  fun handle(artifact: DeliveryArtifact)
+}
```

Comment:
New interface I'll use for checking artifacts on a schedule. `ImageHandler` will become one of these instead of a `ResourceHandler`.